### ISSUE TITLE
Flood of renewing subscription Errors

### ISSUE THREAD

**Issue by badabing2005** [fwestenberg/reolink_dev#110]
![image](https://user-images.githubusercontent.com/22603312/102354135-2556c100-3f78-11eb-8dae-85d4dfd1eec6.png)

I originally posted the issue here:
_Originally posted by @badabing2005 in https://github.com/fwestenberg/reolink_dev/issues/94#issuecomment-746281509_

That issue being closed, I'm logging this as a new issue so that we can keep track of it.

I enabled debug logging and managed to capture a bit more details though I'm not sure if it is enough.
It looks like the http response code is 400  Bad request?
Perhaps the request in case of NVR is not correct?

```
2020-12-20 08:20:07 ERROR (MainThread) [custom_components.reolink_dev.base] Host 192.168.1.100 error renewing the Reolink subscription

2020-12-20 08:20:07 DEBUG (MainThread) [reolink.subscription_manager] Reolink host 192.168.1.100 got response status: 400. Payload: {<?xml version="1.0" encoding="UTF-8"?>
<SOAP-ENV:Envelope xmlns:SOAP-ENV="http://www.w3.org/2003/05/soap-envelope" xmlns:SOAP-ENC="http://www.w3.org/2003/05/soap-encoding" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:wsa="http://schemas.xmlsoap.org/ws/2004/08/addressing" xmlns:wsdd="http://schemas.xmlsoap.org/ws/2005/04/discovery" xmlns:wsa5="http://www.w3.org/2005/08/addressing" xmlns:xmime="http://tempuri.org/xmime.xsd" xmlns:xmime5="http://www.w3.org/2005/05/xmlmime" xmlns:xop="http://www.w3.org/2004/08/xop/include" xmlns:wsrfbf="http://docs.oasis-open.org/wsrf/bf-2" xmlns:tt="http://www.onvif.org/ver10/schema" xmlns:wstop="http://docs.oasis-open.org/wsn/t-1" xmlns:wsrfr="http://docs.oasis-open.org/wsrf/r-2" xmlns:ns1="http://www.onvif.org/ver10/actionengine/wsdl" xmlns:tev="http://www.onvif.org/ver10/events/wsdl" xmlns:ns10="http://www.onvif.org/ver10/events/wsdl/PullPointBinding" xmlns:ns11="http://www.onvif.org/ver10/events/wsdl/CreatePullPointBinding" xmlns:ns12="http://www.onvif.org/ver10/events/wsdl/PausableSubscriptionManagerBinding" xmlns:ns13="http://www.onvif.org/ver10/network/wsdl/RemoteDiscoveryBinding" xmlns:ns14="http://www.onvif.org/ver10/network/wsdl/DiscoveryLookupBinding" xmlns:tdn="http://www.onvif.org/ver10/network/wsdl" xmlns:ns3="http://www.onvif.org/ver20/analytics/wsdl/RuleEngineBinding" xmlns:ns4="http://www.onvif.org/ver20/analytics/wsdl/AnalyticsEngineBinding" xmlns:tan="http://www.onvif.org/ver20/analytics/wsdl" xmlns:ns5="http://www.onvif.org/ver10/events/wsdl/PullPointSubscriptionBinding" xmlns:ns6="http://www.onvif.org/ver10/events/wsdl/EventBinding" xmlns:ns7="http://www.onvif.org/ver10/events/wsdl/SubscriptionManagerBinding" xmlns:ns8="http://www.onvif.org/ver10/events/wsdl/NotificationProducerBinding" xmlns:wsnt="http://docs.oasis-open.org/wsn/b-2" xmlns:ns9="http://www.onvif.org/ver10/events/wsdl/NotificationConsumerBinding" xmlns:tad="http://www.onvif.org/ver10/analyticsdevice/wsdl" xmlns:tds="http://www.onvif.org/ver10/device/wsdl" xmlns:timg="http://www.onvif.org/ver20/imaging/wsdl" xmlns:tls="http://www.onvif.org/ver10/display/wsdl" xmlns:tmd="http://www.onvif.org/ver10/deviceIO/wsdl" xmlns:tptz="http://www.onvif.org/ver20/ptz/wsdl" xmlns:trc="http://www.onvif.org/ver10/recording/wsdl" xmlns:trp="http://www.onvif.org/ver10/replay/wsdl" xmlns:trt="http://www.onvif.org/ver10/media/wsdl" xmlns:trv="http://www.onvif.org/ver10/receiver/wsdl" xmlns:tse="http://www.onvif.org/ver10/search/wsdl" xmlns:ter="http://www.onvif.org/ver10/error" xmlns:tns1="http://www.onvif.org/ver10/topics" xmlns:wsse="http://docs.oasis-open.org/wss/2004/01/oasis-200401-wss-wssecurity-secext-1.0.xsd" xmlns:wsu="http://docs.oasis-open.org/wss/2004/01/oasis-200401-wss-wssecurity-utility-1.0.xsd"><SOAP-ENV:Header></SOAP-ENV:Header><SOAP-ENV:Body><SOAP-ENV:Fault><SOAP-ENV:Code><SOAP-ENV:Value>SOAP-ENV:Sender</SOAP-ENV:Value></SOAP-ENV:Code><SOAP-ENV:Reason><SOAP-ENV:Text xml:lang="en"/></SOAP-ENV:Reason></SOAP-ENV:Fault></SOAP-ENV:Body></SOAP-ENV:Envelope>
}

2020-12-20 08:20:07 DEBUG (MainThread) [custom_components.reolink_dev] Finished fetching reolink data in 0.305 seconds
```

**Comment by fwestenberg** [comment]
Error logs should be gone now. Right?

**Comment by badabing2005** [comment]
Unfortunately no,
The minute I added more cameras they came back,

```
2020-12-21 17:33:47 DEBUG (MainThread) [reolink.subscription_manager] Reolink host 192.168.1.100 got response status: 400. Payload: {<?xml version="1.0" encoding="UTF-8"?>
<SOAP-ENV:Envelope xmlns:SOAP-ENV="http://www.w3.org/2003/05/soap-envelope" xmlns:SOAP-ENC="http://www.w3.org/2003/05/soap-encoding" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:wsa="http://schemas.xmlsoap.org/ws/2004/08/addressing" xmlns:wsdd="http://schemas.xmlsoap.org/ws/2005/04/discovery" xmlns:wsa5="http://www.w3.org/2005/08/addressing" xmlns:xmime="http://tempuri.org/xmime.xsd" xmlns:xmime5="http://www.w3.org/2005/05/xmlmime" xmlns:xop="http://www.w3.org/2004/08/xop/include" xmlns:wsrfbf="http://docs.oasis-open.org/wsrf/bf-2" xmlns:tt="http://www.onvif.org/ver10/schema" xmlns:wstop="http://docs.oasis-open.org/wsn/t-1" xmlns:wsrfr="http://docs.oasis-open.org/wsrf/r-2" xmlns:ns1="http://www.onvif.org/ver10/actionengine/wsdl" xmlns:tev="http://www.onvif.org/ver10/events/wsdl" xmlns:ns10="http://www.onvif.org/ver10/events/wsdl/PullPointBinding" xmlns:ns11="http://www.onvif.org/ver10/events/wsdl/CreatePullPointBinding" xmlns:ns12="http://www.onvif.org/ver10/events/wsdl/PausableSubscriptionManagerBinding" xmlns:ns13="http://www.onvif.org/ver10/network/wsdl/RemoteDiscoveryBinding" xmlns:ns14="http://www.onvif.org/ver10/network/wsdl/DiscoveryLookupBinding" xmlns:tdn="http://www.onvif.org/ver10/network/wsdl" xmlns:ns3="http://www.onvif.org/ver20/analytics/wsdl/RuleEngineBinding" xmlns:ns4="http://www.onvif.org/ver20/analytics/wsdl/AnalyticsEngineBinding" xmlns:tan="http://www.onvif.org/ver20/analytics/wsdl" xmlns:ns5="http://www.onvif.org/ver10/events/wsdl/PullPointSubscriptionBinding" xmlns:ns6="http://www.onvif.org/ver10/events/wsdl/EventBinding" xmlns:ns7="http://www.onvif.org/ver10/events/wsdl/SubscriptionManagerBinding" xmlns:ns8="http://www.onvif.org/ver10/events/wsdl/NotificationProducerBinding" xmlns:wsnt="http://docs.oasis-open.org/wsn/b-2" xmlns:ns9="http://www.onvif.org/ver10/events/wsdl/NotificationConsumerBinding" xmlns:tad="http://www.onvif.org/ver10/analyticsdevice/wsdl" xmlns:tds="http://www.onvif.org/ver10/device/wsdl" xmlns:timg="http://www.onvif.org/ver20/imaging/wsdl" xmlns:tls="http://www.onvif.org/ver10/display/wsdl" xmlns:tmd="http://www.onvif.org/ver10/deviceIO/wsdl" xmlns:tptz="http://www.onvif.org/ver20/ptz/wsdl" xmlns:trc="http://www.onvif.org/ver10/recording/wsdl" xmlns:trp="http://www.onvif.org/ver10/replay/wsdl" xmlns:trt="http://www.onvif.org/ver10/media/wsdl" xmlns:trv="http://www.onvif.org/ver10/receiver/wsdl" xmlns:tse="http://www.onvif.org/ver10/search/wsdl" xmlns:ter="http://www.onvif.org/ver10/error" xmlns:tns1="http://www.onvif.org/ver10/topics" xmlns:wsse="http://docs.oasis-open.org/wss/2004/01/oasis-200401-wss-wssecurity-secext-1.0.xsd" xmlns:wsu="http://docs.oasis-open.org/wss/2004/01/oasis-200401-wss-wssecurity-utility-1.0.xsd"><SOAP-ENV:Header><wsa5:To SOAP-ENV:mustUnderstand="1">None</wsa5:To><wsa5:Action SOAP-ENV:mustUnderstand="1">http://docs.oasis-open.org/wsn/bw-2/SubscriptionManager/RenewRequest</wsa5:Action></SOAP-ENV:Header><SOAP-ENV:Body><SOAP-ENV:Fault><SOAP-ENV:Code><SOAP-ENV:Value>SOAP-ENV:Sender</SOAP-ENV:Value></SOAP-ENV:Code><SOAP-ENV:Reason><SOAP-ENV:Text xml:lang="en"/></SOAP-ENV:Reason></SOAP-ENV:Fault></SOAP-ENV:Body></SOAP-ENV:Envelope>
}
2020-12-21 17:33:47 DEBUG (MainThread) [reolink.subscription_manager] Reolink host 192.168.1.100 got response status: 400. Payload: {<?xml version="1.0" encoding="UTF-8"?>
<SOAP-ENV:Envelope xmlns:SOAP-ENV="http://www.w3.org/2003/05/soap-envelope" xmlns:SOAP-ENC="http://www.w3.org/2003/05/soap-encoding" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:wsa="http://schemas.xmlsoap.org/ws/2004/08/addressing" xmlns:wsdd="http://schemas.xmlsoap.org/ws/2005/04/discovery" xmlns:wsa5="http://www.w3.org/2005/08/addressing" xmlns:xmime="http://tempuri.org/xmime.xsd" xmlns:xmime5="http://www.w3.org/2005/05/xmlmime" xmlns:xop="http://www.w3.org/2004/08/xop/include" xmlns:wsrfbf="http://docs.oasis-open.org/wsrf/bf-2" xmlns:tt="http://www.onvif.org/ver10/schema" xmlns:wstop="http://docs.oasis-open.org/wsn/t-1" xmlns:wsrfr="http://docs.oasis-open.org/wsrf/r-2" xmlns:ns1="http://www.onvif.org/ver10/actionengine/wsdl" xmlns:tev="http://www.onvif.org/ver10/events/wsdl" xmlns:ns10="http://www.onvif.org/ver10/events/wsdl/PullPointBinding" xmlns:ns11="http://www.onvif.org/ver10/events/wsdl/CreatePullPointBinding" xmlns:ns12="http://www.onvif.org/ver10/events/wsdl/PausableSubscriptionManagerBinding" xmlns:ns13="http://www.onvif.org/ver10/network/wsdl/RemoteDiscoveryBinding" xmlns:ns14="http://www.onvif.org/ver10/network/wsdl/DiscoveryLookupBinding" xmlns:tdn="http://www.onvif.org/ver10/network/wsdl" xmlns:ns3="http://www.onvif.org/ver20/analytics/wsdl/RuleEngineBinding" xmlns:ns4="http://www.onvif.org/ver20/analytics/wsdl/AnalyticsEngineBinding" xmlns:tan="http://www.onvif.org/ver20/analytics/wsdl" xmlns:ns5="http://www.onvif.org/ver10/events/wsdl/PullPointSubscriptionBinding" xmlns:ns6="http://www.onvif.org/ver10/events/wsdl/EventBinding" xmlns:ns7="http://www.onvif.org/ver10/events/wsdl/SubscriptionManagerBinding" xmlns:ns8="http://www.onvif.org/ver10/events/wsdl/NotificationProducerBinding" xmlns:wsnt="http://docs.oasis-open.org/wsn/b-2" xmlns:ns9="http://www.onvif.org/ver10/events/wsdl/NotificationConsumerBinding" xmlns:tad="http://www.onvif.org/ver10/analyticsdevice/wsdl" xmlns:tds="http://www.onvif.org/ver10/device/wsdl" xmlns:timg="http://www.onvif.org/ver20/imaging/wsdl" xmlns:tls="http://www.onvif.org/ver10/display/wsdl" xmlns:tmd="http://www.onvif.org/ver10/deviceIO/wsdl" xmlns:tptz="http://www.onvif.org/ver20/ptz/wsdl" xmlns:trc="http://www.onvif.org/ver10/recording/wsdl" xmlns:trp="http://www.onvif.org/ver10/replay/wsdl" xmlns:trt="http://www.onvif.org/ver10/media/wsdl" xmlns:trv="http://www.onvif.org/ver10/receiver/wsdl" xmlns:tse="http://www.onvif.org/ver10/search/wsdl" xmlns:ter="http://www.onvif.org/ver10/error" xmlns:tns1="http://www.onvif.org/ver10/topics" xmlns:wsse="http://docs.oasis-open.org/wss/2004/01/oasis-200401-wss-wssecurity-secext-1.0.xsd" xmlns:wsu="http://docs.oasis-open.org/wss/2004/01/oasis-200401-wss-wssecurity-utility-1.0.xsd"><SOAP-ENV:Header><wsa5:To SOAP-ENV:mustUnderstand="1">None</wsa5:To><wsa5:Action SOAP-ENV:mustUnderstand="1">http://docs.oasis-open.org/wsn/bw-2/SubscriptionManager/UnsubscribeRequest</wsa5:Action></SOAP-ENV:Header><SOAP-ENV:Body><SOAP-ENV:Fault><SOAP-ENV:Code><SOAP-ENV:Value>SOAP-ENV:Sender</SOAP-ENV:Value></SOAP-ENV:Code><SOAP-ENV:Reason><SOAP-ENV:Text xml:lang="en"/></SOAP-ENV:Reason></SOAP-ENV:Fault></SOAP-ENV:Body></SOAP-ENV:Envelope>
}
2020-12-21 17:33:47 ERROR (MainThread) [custom_components.reolink_dev.base] Host 192.168.1.100 error renewing the Reolink subscription
2020-12-21 17:33:47 DEBUG (MainThread) [reolink.subscription_manager] Reolink host 192.168.1.100 got response status: 400. Payload: {<?xml version="1.0" encoding="UTF-8"?>
<SOAP-ENV:Envelope xmlns:SOAP-ENV="http://www.w3.org/2003/05/soap-envelope" xmlns:SOAP-ENC="http://www.w3.org/2003/05/soap-encoding" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:wsa="http://schemas.xmlsoap.org/ws/2004/08/addressing" xmlns:wsdd="http://schemas.xmlsoap.org/ws/2005/04/discovery" xmlns:wsa5="http://www.w3.org/2005/08/addressing" xmlns:xmime="http://tempuri.org/xmime.xsd" xmlns:xmime5="http://www.w3.org/2005/05/xmlmime" xmlns:xop="http://www.w3.org/2004/08/xop/include" xmlns:wsrfbf="http://docs.oasis-open.org/wsrf/bf-2" xmlns:tt="http://www.onvif.org/ver10/schema" xmlns:wstop="http://docs.oasis-open.org/wsn/t-1" xmlns:wsrfr="http://docs.oasis-open.org/wsrf/r-2" xmlns:ns1="http://www.onvif.org/ver10/actionengine/wsdl" xmlns:tev="http://www.onvif.org/ver10/events/wsdl" xmlns:ns10="http://www.onvif.org/ver10/events/wsdl/PullPointBinding" xmlns:ns11="http://www.onvif.org/ver10/events/wsdl/CreatePullPointBinding" xmlns:ns12="http://www.onvif.org/ver10/events/wsdl/PausableSubscriptionManagerBinding" xmlns:ns13="http://www.onvif.org/ver10/network/wsdl/RemoteDiscoveryBinding" xmlns:ns14="http://www.onvif.org/ver10/network/wsdl/DiscoveryLookupBinding" xmlns:tdn="http://www.onvif.org/ver10/network/wsdl" xmlns:ns3="http://www.onvif.org/ver20/analytics/wsdl/RuleEngineBinding" xmlns:ns4="http://www.onvif.org/ver20/analytics/wsdl/AnalyticsEngineBinding" xmlns:tan="http://www.onvif.org/ver20/analytics/wsdl" xmlns:ns5="http://www.onvif.org/ver10/events/wsdl/PullPointSubscriptionBinding" xmlns:ns6="http://www.onvif.org/ver10/events/wsdl/EventBinding" xmlns:ns7="http://www.onvif.org/ver10/events/wsdl/SubscriptionManagerBinding" xmlns:ns8="http://www.onvif.org/ver10/events/wsdl/NotificationProducerBinding" xmlns:wsnt="http://docs.oasis-open.org/wsn/b-2" xmlns:ns9="http://www.onvif.org/ver10/events/wsdl/NotificationConsumerBinding" xmlns:tad="http://www.onvif.org/ver10/analyticsdevice/wsdl" xmlns:tds="http://www.onvif.org/ver10/device/wsdl" xmlns:timg="http://www.onvif.org/ver20/imaging/wsdl" xmlns:tls="http://www.onvif.org/ver10/display/wsdl" xmlns:tmd="http://www.onvif.org/ver10/deviceIO/wsdl" xmlns:tptz="http://www.onvif.org/ver20/ptz/wsdl" xmlns:trc="http://www.onvif.org/ver10/recording/wsdl" xmlns:trp="http://www.onvif.org/ver10/replay/wsdl" xmlns:trt="http://www.onvif.org/ver10/media/wsdl" xmlns:trv="http://www.onvif.org/ver10/receiver/wsdl" xmlns:tse="http://www.onvif.org/ver10/search/wsdl" xmlns:ter="http://www.onvif.org/ver10/error" xmlns:tns1="http://www.onvif.org/ver10/topics" xmlns:wsse="http://docs.oasis-open.org/wss/2004/01/oasis-200401-wss-wssecurity-secext-1.0.xsd" xmlns:wsu="http://docs.oasis-open.org/wss/2004/01/oasis-200401-wss-wssecurity-utility-1.0.xsd"><SOAP-ENV:Header></SOAP-ENV:Header><SOAP-ENV:Body><SOAP-ENV:Fault><SOAP-ENV:Code><SOAP-ENV:Value>SOAP-ENV:Sender</SOAP-ENV:Value></SOAP-ENV:Code><SOAP-ENV:Reason><SOAP-ENV:Text xml:lang="en"/></SOAP-ENV:Reason></SOAP-ENV:Fault></SOAP-ENV:Body></SOAP-ENV:Envelope>
}
2020-12-21 17:33:47 DEBUG (MainThread) [custom_components.reolink_dev] Finished fetching reolink data in 0.291 seconds
2020-12-21 17:34:05 DEBUG (MainThread) [reolink.subscription_manager] Reolink host 192.168.1.100 got response status: 400. Payload: {<?xml version="1.0" encoding="UTF-8"?>
<SOAP-ENV:Envelope xmlns:SOAP-ENV="http://www.w3.org/2003/05/soap-envelope" xmlns:SOAP-ENC="http://www.w3.org/2003/05/soap-encoding" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:wsa="http://schemas.xmlsoap.org/ws/2004/08/addressing" xmlns:wsdd="http://schemas.xmlsoap.org/ws/2005/04/discovery" xmlns:wsa5="http://www.w3.org/2005/08/addressing" xmlns:xmime="http://tempuri.org/xmime.xsd" xmlns:xmime5="http://www.w3.org/2005/05/xmlmime" xmlns:xop="http://www.w3.org/2004/08/xop/include" xmlns:wsrfbf="http://docs.oasis-open.org/wsrf/bf-2" xmlns:tt="http://www.onvif.org/ver10/schema" xmlns:wstop="http://docs.oasis-open.org/wsn/t-1" xmlns:wsrfr="http://docs.oasis-open.org/wsrf/r-2" xmlns:ns1="http://www.onvif.org/ver10/actionengine/wsdl" xmlns:tev="http://www.onvif.org/ver10/events/wsdl" xmlns:ns10="http://www.onvif.org/ver10/events/wsdl/PullPointBinding" xmlns:ns11="http://www.onvif.org/ver10/events/wsdl/CreatePullPointBinding" xmlns:ns12="http://www.onvif.org/ver10/events/wsdl/PausableSubscriptionManagerBinding" xmlns:ns13="http://www.onvif.org/ver10/network/wsdl/RemoteDiscoveryBinding" xmlns:ns14="http://www.onvif.org/ver10/network/wsdl/DiscoveryLookupBinding" xmlns:tdn="http://www.onvif.org/ver10/network/wsdl" xmlns:ns3="http://www.onvif.org/ver20/analytics/wsdl/RuleEngineBinding" xmlns:ns4="http://www.onvif.org/ver20/analytics/wsdl/AnalyticsEngineBinding" xmlns:tan="http://www.onvif.org/ver20/analytics/wsdl" xmlns:ns5="http://www.onvif.org/ver10/events/wsdl/PullPointSubscriptionBinding" xmlns:ns6="http://www.onvif.org/ver10/events/wsdl/EventBinding" xmlns:ns7="http://www.onvif.org/ver10/events/wsdl/SubscriptionManagerBinding" xmlns:ns8="http://www.onvif.org/ver10/events/wsdl/NotificationProducerBinding" xmlns:wsnt="http://docs.oasis-open.org/wsn/b-2" xmlns:ns9="http://www.onvif.org/ver10/events/wsdl/NotificationConsumerBinding" xmlns:tad="http://www.onvif.org/ver10/analyticsdevice/wsdl" xmlns:tds="http://www.onvif.org/ver10/device/wsdl" xmlns:timg="http://www.onvif.org/ver20/imaging/wsdl" xmlns:tls="http://www.onvif.org/ver10/display/wsdl" xmlns:tmd="http://www.onvif.org/ver10/deviceIO/wsdl" xmlns:tptz="http://www.onvif.org/ver20/ptz/wsdl" xmlns:trc="http://www.onvif.org/ver10/recording/wsdl" xmlns:trp="http://www.onvif.org/ver10/replay/wsdl" xmlns:trt="http://www.onvif.org/ver10/media/wsdl" xmlns:trv="http://www.onvif.org/ver10/receiver/wsdl" xmlns:tse="http://www.onvif.org/ver10/search/wsdl" xmlns:ter="http://www.onvif.org/ver10/error" xmlns:tns1="http://www.onvif.org/ver10/topics" xmlns:wsse="http://docs.oasis-open.org/wss/2004/01/oasis-200401-wss-wssecurity-secext-1.0.xsd" xmlns:wsu="http://docs.oasis-open.org/wss/2004/01/oasis-200401-wss-wssecurity-utility-1.0.xsd"><SOAP-ENV:Header><wsa5:To SOAP-ENV:mustUnderstand="1">None</wsa5:To><wsa5:Action SOAP-ENV:mustUnderstand="1">http://docs.oasis-open.org/wsn/bw-2/SubscriptionManager/RenewRequest</wsa5:Action></SOAP-ENV:Header><SOAP-ENV:Body><SOAP-ENV:Fault><SOAP-ENV:Code><SOAP-ENV:Value>SOAP-ENV:Sender</SOAP-ENV:Value></SOAP-ENV:Code><SOAP-ENV:Reason><SOAP-ENV:Text xml:lang="en"/></SOAP-ENV:Reason></SOAP-ENV:Fault></SOAP-ENV:Body></SOAP-ENV:Envelope>
}
```

It might be related to a device mixup
Adding the first camera, the device id is properly picked up from the camera name
then I proceeded to add more cameras

Notice what happens
The last camera that I added was `Tempo`
![image](https://user-images.githubusercontent.com/22603312/102828915-dbc10880-43b3-11eb-9735-1386ea2d5a1e.png)

So let's dive into the details of Driveway camera
![image](https://user-images.githubusercontent.com/22603312/102828945-f1cec900-43b3-11eb-80ef-f6ba265854cb.png)
The entities are all correct, they properly reflect the camera name in the entity names
However the device id is wrong

notice how clicking on it opens up `Tempo` device and not `Driveway` device
![image](https://user-images.githubusercontent.com/22603312/102829022-204ca400-43b4-11eb-854b-15db7b4e2ee5.png)

This is the case with all the cameras
so it looks like the last camera device id is overwriting all camera device ids,
perhaps why subscription is failing (just a guess)

Before adding multiple cameras, I had no issues with subscription errors.

Hope this helps

**Comment by fwestenberg** [comment]
From what version did you notice this issue? Can you remember?

**Comment by badabing2005** [comment]
> From what version did you notice this issue? Can you remember?

0.9

**Comment by kaaspad** [comment]
I'm having the exact same issue on 0.9 (0.8 had more issues where the naming was wrong) - so actually the same two issues which I don't know if they are related to each other:
1. I have 6 cameras, but similarly to @badabing2005 they all have the correct entities, but each camera (channel) added overrides the device of the previous one, which means by me all the devices are called "Porch"
2. Getting log flooded with: `ERROR (MainThread) [custom_components.reolink_dev.base] Host 192.xxx.xxx.xxx error renewing the Reolink subscription`

**Comment by kaaspad** [comment]
any updates here? still getting a flood of messages as well as the issue with multiple devices with same name on NVR? no pressure though.... :-)

**Comment by RikyUnreal** [comment]
I have the same _error renewing the Reolink subscription_ issue; no pressure, just for your information.
My equipment: NRV RLN8-410-E

**Comment by fwestenberg** [comment]
I renamed this issue a bit. To be clear for this issue:
- Initial subscription works well
- Subscription manager counts down
- Renew goes wrong: http 400 message

If the initial subscription gives http 400: please post in issue #115

**Comment by RikyUnreal** [comment]
Just updated to 0.13
Same issue
 ```
(Logger: custom_components.reolink_dev.base
Source: custom_components/reolink_dev/base.py:24)
```
My equipment: NRV RLN8-410-E

**Comment by fwestenberg** [comment]
Did the integration subscribe at all? Or did you get http 400 from the start? If this is the case, the NVR probably has already too many subscriptions. This should be solved after +- 15 minutes.

**Comment by RikyUnreal** [comment]
Where I should get the 400 error?
Now I simply check the HA log and see "message first occurred at 12:02:57 PM and shows up 11 times".... and counting

**Comment by fwestenberg** [comment]
Just wait 15min and go to the integration. There is a option 'reload' in the menu. After that, check the log. I think this should fix it.

**Comment by RikyUnreal** [comment]
Ok waiting and I'll give you an update.
I suppose I have to reload all cameras right?

**Comment by cob94440** [comment]
Just did the reload and still same error.

**Comment by cob94440** [comment]
Logger: custom_components.reolink_dev.base
Source: custom_components/reolink_dev/base.py:241
Integration: Reolink IP camera (documentation, issues)
First occurred: 12:25:14 (8 occurrences)
Last logged: 12:26:16

Host 10.0.0.36 error renewing the Reolink subscription

**Comment by RikyUnreal** [comment]
Waited, reloaded all cameras, but same issue.

**Comment by fwestenberg** [comment]
Post the log please! From the moment you reloaded the integration.

**Comment by RikyUnreal** [comment]
```
Logger: custom_components.reolink_dev.base
Source: custom_components/reolink_dev/base.py:241
Integration: Reolink IP camera (documentation, issues)
First occurred: 12:28:57 PM (29 occurrences)
Last logged: 12:35:23 PM
```

Reloaded at 12:28

**Comment by cob94440** [comment]
2021-01-17 12:30:19 ERROR (MainThread) [homeassistant.components.scrape.sensor] Unable to retrieve data for hacf_essence_cheroy_inter_SP98_price
2021-01-17 12:30:20 ERROR (MainThread) [custom_components.reolink_dev.base] Host 10.0.0.36 error renewing the Reolink subscription
2021-01-17 12:30:20 ERROR (MainThread) [custom_components.reolink_dev.base] Host 10.0.0.36 error renewing the Reolink subscription
2021-01-17 12:30:24 WARNING (MainThread) [homeassistant.helpers.entity] Update of sensor.chris_to_papy is taking over 10 seconds
2021-01-17 12:30:43 WARNING (MainThread) [homeassistant.components.sensor] Updating badnest sensor took longer than the scheduled update interval 0:00:30
2021-01-17 12:31:20 ERROR (MainThread) [custom_components.reolink_dev.base] Host 10.0.0.36 error renewing the Reolink subscription
2021-01-17 12:31:21 ERROR (MainThread) [custom_components.reolink_dev.base] Host 10.0.0.36 error renewing the Reolink subscription
2021-01-17 12:31:33 WARNING (MainThread) [homeassistant.components.webhook] Received message for unregistered webhook b3212d801317713773e8d7748aad49b17d8b7abaf247376e261cfad592dbb647 from 10.0.0.36
2021-01-17 12:31:36 WARNING (MainThread) [homeassistant.components.webhook] Received message for unregistered webhook b3212d801317713773e8d7748aad49b17d8b7abaf247376e261cfad592dbb647 from 10.0.0.36
2021-01-17 12:31:40 WARNING (MainThread) [homeassistant.components.webhook] Received message for unregistered webhook b3212d801317713773e8d7748aad49b17d8b7abaf247376e261cfad592dbb647 from 10.0.0.36
2021-01-17 12:31:43 WARNING (MainThread) [homeassistant.components.sensor] Updating badnest sensor took longer than the scheduled update interval 0:00:30
2021-01-17 12:31:45 WARNING (MainThread) [homeassistant.components.webhook] Received message for unregistered webhook b3212d801317713773e8d7748aad49b17d8b7abaf247376e261cfad592dbb647 from 10.0.0.36
2021-01-17 12:32:01 WARNING (MainThread) [homeassistant.components.webhook] Received message for unregistered webhook b3212d801317713773e8d7748aad49b17d8b7abaf247376e261cfad592dbb647 from 10.0.0.36
2021-01-17 12:32:04 WARNING (MainThread) [homeassistant.components.webhook] Received message for unregistered webhook b3212d801317713773e8d7748aad49b17d8b7abaf247376e261cfad592dbb647 from 10.0.0.36
2021-01-17 12:32:09 WARNING (MainThread) [homeassistant.components.webhook] Received message for unregistered webhook b3212d801317713773e8d7748aad49b17d8b7abaf247376e261cfad592dbb647 from 10.0.0.36
2021-01-17 12:32:12 WARNING (MainThread) [homeassistant.components.webhook] Received message for unregistered webhook b3212d801317713773e8d7748aad49b17d8b7abaf247376e261cfad592dbb647 from 10.0.0.36
2021-01-17 12:32:16 WARNING (MainThread) [homeassistant.components.webhook] Received message for unregistered webhook b3212d801317713773e8d7748aad49b17d8b7abaf247376e261cfad592dbb647 from 10.0.0.36
2021-01-17 12:32:20 ERROR (MainThread) [custom_components.reolink_dev.base] Host 10.0.0.36 error renewing the Reolink subscription
2021-01-17 12:32:21 WARNING (MainThread) [homeassistant.components.webhook] Received message for unregistered webhook b3212d801317713773e8d7748aad49b17d8b7abaf247376e261cfad592dbb647 from 10.0.0.36
2021-01-17 12:32:21 ERROR (MainThread) [custom_components.reolink_dev.base] Host 10.0.0.36 error renewing the Reolink subscription
2021-01-17 12:32:43 WARNING (MainThread) [homeassistant.components.sensor] Updating badnest sensor took longer than the scheduled update interval 0:00:30
2021-01-17 12:33:20 ERROR (MainThread) [custom_components.reolink_dev.base] Host 10.0.0.36 error renewing the Reolink subscription
2021-01-17 12:33:21 ERROR (MainThread) [custom_components.reolink_dev.base] Host 10.0.0.36 error renewing the Reolink subscription
2021-01-17 12:33:43 WARNING (MainThread) [homeassistant.components.sensor] Updating badnest sensor took longer than the scheduled update interval 0:00:30
2021-01-17 12:34:20 ERROR (MainThread) [custom_components.reolink_dev.base] Host 10.0.0.36 error renewing the Reolink subscription
2021-01-17 12:34:21 ERROR (MainThread) [custom_components.reolink_dev.base] Host 10.0.0.36 error renewing the Reolink subscription
2021-01-17 12:35:13 WARNING (MainThread) [homeassistant.components.sensor] Updating badnest sensor took longer than the scheduled update interval 0:00:30
2021-01-17 12:35:19 WARNING (MainThread) [homeassistant.helpers.entity] Update of sensor.palier_protect_battery_health_state is taking over 10 seconds
2021-01-17 12:35:21 ERROR (MainThread) [custom_components.reolink_dev.base] Host 10.0.0.36 error renewing the Reolink subscription
2021-01-17 12:36:05 WARNING (MainThread) [homeassistant.helpers.entity] Update of sensor.kitchen_protect_battery_health_state is taking over 10 seconds
2021-01-17 12:36:13 WARNING (MainThread) [homeassistant.components.sensor] Updating badnest sensor took longer than the scheduled update interval 0:00:30
2021-01-17 12:36:21 ERROR (MainThread) [custom_components.reolink_dev.base] Host 10.0.0.36 error renewing the Reolink subscription
2021-01-17 12:37:21 ERROR (MainThread) [custom_components.reolink_dev.base] Host 10.0.0.36 error renewing the Reolink subscription

**Comment by fwestenberg** [comment]
in the logging config, you probably miss the item:
`reolink: debug`

But it looks like the NVS is still calling the old webhook, so this subscription is still alive. A reboot of the NVR will fix this I guess. The integration should then reconnect by itself within a minute.

**Comment by RikyUnreal** [comment]
My configuration.yaml
```
logger:
  logs:
    custom_components.reolink_dev: debug
    reolink: debug
```

Now turning OFF the NVR and rebooting.

**Comment by cob94440** [comment]
same after rebooting nvr

**Comment by RikyUnreal** [comment]
After rebooting the NVR, I also removed all cameras.
Now I added only one camera, and it seems I get **no errors**.
Waiting a bit before adding the other cameras

**Comment by RikyUnreal** [comment]
Added also the 2nd and 3rd cameras, and still no renewing no errors. 👍 
Waiting a bit before adding the last one.

**Comment by cob94440** [comment]
i deleted camerafrom ha. rebooted nvr. rebooted ha
added camera 1 ok
camera 2 still no message. but ftp shows on for camera 2 which is not true
idem camera 3
camera 4 subscription  error is back 

Logger: custom_components.reolink_dev.base
Source: custom_components/reolink_dev/base.py:241
Integration: Reolink IP camera (documentation, issues)
First occurred: 14:08:56 (1 occurrences)
Last logged: 14:08:56

Host 10.0.0.36 error renewing the Reolink subscription


motion then unavailable for all camera

Ftp seems on for camera 2,3 and 4 unable to disable

**Comment by RikyUnreal** [comment]
Same here.
Added camera 4 and get the error back

**Comment by cob94440** [comment]
> Same here.
> Added camera 4 and get the error back

Do you also have ftp on for camera in ha?

**Comment by RikyUnreal** [comment]
> Do you also have ftp on for camera in ha?
I don't use FTP feature on the NVR so I never tried to activate/deactivate it from HA

**Comment by cob94440** [comment]
ok but do it show on for camera 2,3 and 4. I do not use it either

**Comment by cob94440** [comment]
Seems like i don't have the latest firmware like you.
![image](https://user-images.githubusercontent.com/62489067/104845481-08ded900-58d6-11eb-9657-aae9b081b023.png)

**Comment by cob94440** [comment]
Now ftp is ok on latest version.
I uncheck on nvr the triggered that record all channels.
But still no motion.
And still subscription problem with 4 camera

**Comment by jgthornburgh** [comment]
I'm having the same issue after updating to v.13. All motion binary sensors are unavailable.

**Comment by RikyUnreal** [comment]
Just updated to 0.14
Some news: I get some error after  HA reload, but then the log seem stable.

My equipment: NRV RLN8-410-E and 4 camera installed

(Logger: custom_components.reolink_dev.base
Source: custom_components/reolink_dev/base.py:245)

1. First "batch of errors **400**, several time
`2021-02-16 18:51:58 ERROR (MainThread) [custom_components.reolink_dev.base] Host 192.168.178.51 error renewing the Reolink subscription
2021-02-16 18:51:58 DEBUG (MainThread) [homeassistant.core] Bus:Handling <Event reolink_dev-event-EC71DB1814B3[L]: available=False>
2021-02-16 18:51:58 DEBUG (MainThread) [reolink.subscription_manager] Reolink host 192.168.178.51 got response status: 400. Payload: {<?xml version="1.0" encoding="UTF-8"?>`

2. Then **200**
`2021-02-16 18:51:59 ERROR (MainThread) [custom_components.reolink_dev.base] Host 192.168.178.51 error renewing the Reolink subscription
2021-02-16 18:51:59 DEBUG (MainThread) [homeassistant.core] Bus:Handling <Event reolink_dev-event-EC71DB1814B3[L]: available=False>
2021-02-16 18:51:59 DEBUG (MainThread) [reolink.subscription_manager] Reolink host 192.168.178.51 got response status: 200. Payload: {<?xml version="1.0" encoding="UTF-8"?>`

3. Then this and no more errors in the log
`2021-02-16 18:53:58 DEBUG (MainThread) [reolink.subscription_manager] Host 192.168.178.51 should renew in: 781 seconds...
2021-02-16 18:53:58 DEBUG (MainThread) [homeassistant.core] Bus:Handling <Event reolink_dev-event-EC71DB1814B3[L]: available=True>
2021-02-16 18:53:58 DEBUG (MainThread) [reolink.subscription_manager] Host 192.168.178.51 should renew in: 720 seconds...
2021-02-16 18:53:58 DEBUG (MainThread) [homeassistant.core] Bus:Handling <Event reolink_dev-event-EC71DB1814B3[L]: available=True>
2021-02-16 18:53:58 DEBUG (MainThread) [reolink.subscription_manager] Host 192.168.178.51 should renew in: 781 seconds...
2021-02-16 18:53:58 DEBUG (MainThread) [homeassistant.core] Bus:Handling <Event reolink_dev-event-EC71DB1814B3[L]: available=True>
2021-02-16 18:53:58 DEBUG (MainThread) [custom_components.reolink_dev] Finished fetching reolink data in 0.371 seconds
2021-02-16 18:53:58 DEBUG (MainThread) [custom_components.reolink_dev] Finished fetching reolink data in 0.391 seconds
2021-02-16 18:53:58 DEBUG (MainThread) [custom_components.reolink_dev] Finished fetching reolink data in 0.843 seconds
2021-02-16 18:53:58 DEBUG (MainThread) [homeassistant.components.http.auth] Authenticated 192.168.178.122 for /api/error/all using bearer token
2021-02-16 18:53:58 DEBUG (MainThread) [homeassistant.components.http.view] Serving /api/error/all to 192.168.178.122 (auth: True)
2021-02-16 18:53:59 DEBUG (MainThread) [reolink.subscription_manager] Host 192.168.178.51 should renew in: 780 seconds...
2021-02-16 18:53:59 DEBUG (MainThread) [homeassistant.core] Bus:Handling <Event reolink_dev-event-EC71DB1814B3[L]: available=True>`

**Comment by fwestenberg** [comment]
Seems fine, right? How many cams are attached? Or don't you have a NVR?

**Comment by RikyUnreal** [comment]
> Seems fine, right? How many cams are attached? Or don't you have a NVR?

Yes NVR and 4 cameras. 
The log this morning is still clear!

**Comment by fwestenberg** [comment]
Great to hear, thanks!

**Comment by gavyb** [comment]
I just thought I would provide an update on this error. My log is full of these (every minute) but the cameras appear to be working well within HA. It would be nice to clean up the log, so any advice/comment on the error?

**Comment by gavyb** [comment]
@fwestenberg - can you provide an update on the logging issue?

**Comment by fwestenberg** [comment]
Hi @gavyb you could just disable the logging for now. I'm still waiting on an update from Reolink.

**Comment by burton666** [comment]
I have had this problem for a long time, how do I disable logging for the reolink component ?

**Comment by FabioEight** [comment]
Same issue here, just a E1 Zoom configured.

**Comment by danielszilagyi** [comment]
Hello,

I integrated an RLN16-410 with 3 cameras and having the same issue, log flooded with:
 Subscription process ended with wrong HTTP status: 400: Bad Request and
ERROR (MainThread) [custom_components.reolink_dev.base] Host mynvr error renewing the Reolink subscription

So this is an issue that won't be fixed due to a bug on the NVR side? Shall I just lower the loglevel for the integration?

**Comment by attzonko** [comment]
My log is also full of this as well about once per minute, and I only have 2 standalone cameras RLC-511W and a RLC-410W. I have tried the workarounds listed in this issue without success. 

Is this truly confirmed as a Firmware issue with Reolink? If so what is the exact root cause? 

```
2022-01-07 18:58:08 ERROR (stream_worker) [homeassistant.components.stream] Error from stream worker: Error opening stream rtmp://192.168.0.73:1935/bcs/channel0_main.bcs?channel=0&stream=0&user=secret&password=secret
2022-01-07 18:59:04 ERROR (MainThread) [custom_components.reolink_dev.base] Host 192.168.0.73 error renewing the Reolink subscription
2022-01-07 19:00:16 ERROR (MainThread) [custom_components.reolink_dev.base] Host 192.168.0.73 error renewing the Reolink subscription
2022-01-07 19:01:28 ERROR (MainThread) [custom_components.reolink_dev.base] Host 192.168.0.73 error renewing the Reolink subscription
2022-01-07 19:02:40 ERROR (MainThread) [custom_components.reolink_dev.base] Host 192.168.0.73 error renewing the Reolink subscription
2022-01-07 19:03:52 ERROR (MainThread) [custom_components.reolink_dev.base] Host 192.168.0.73 error renewing the Reolink subscription
2022-01-07 19:05:04 ERROR (MainThread) [custom_components.reolink_dev.base] Host 192.168.0.73 error renewing the Reolink subscription
2022-01-07 19:06:16 ERROR (MainThread) [custom_components.reolink_dev.base] Host 192.168.0.73 error renewing the Reolink subscription
2022-01-07 19:07:28 ERROR (MainThread) [custom_components.reolink_dev.base] Host 192.168.0.73 error renewing the Reolink subscription
2022-01-07 19:08:40 ERROR (MainThread) [custom_components.reolink_dev.base] Host 192.168.0.73 error renewing the Reolink subscription
2022-01-07 19:09:52 ERROR (MainThread) [custom_components.reolink_dev.base] Host 192.168.0.73 error renewing the Reolink subscription
2022-01-07 19:11:04 ERROR (MainThread) [custom_components.reolink_dev.base] Host 192.168.0.73 error renewing the Reolink subscription
2022-01-07 19:12:16 ERROR (MainThread) [custom_components.reolink_dev.base] Host 192.168.0.73 error renewing the Reolink subscription
2022-01-07 19:13:28 ERROR (MainThread) [custom_components.reolink_dev.base] Host 192.168.0.73 error renewing the Reolink subscription
2022-01-07 19:14:40 ERROR (MainThread) [custom_components.reolink_dev.base] Host 192.168.0.73 error renewing the Reolink subscription
2022-01-07 19:15:02 ERROR (stream_worker) [libav.rtmp] Cannot open connection tcp://192.168.0.73:1935
2022-01-07 19:15:02 ERROR (stream_worker) [homeassistant.components.stream] Error from stream worker: Error opening stream rtmp://192.168.0.73:1935/bcs/channel0_main.bcs?channel=0&stream=0&user=secret&password=secret
2022-01-07 19:15:52 ERROR (MainThread) [custom_components.reolink_dev.base] Host 192.168.0.73 error renewing the Reolink subscription
2022-01-07 19:17:04 ERROR (MainThread) [custom_components.reolink_dev.base] Host 192.168.0.73 error renewing the Reolink subscription
2022-01-07 19:18:16 ERROR (MainThread) [custom_components.reolink_dev.base] Host 192.168.0.73 error renewing the Reolink subscription
2022-01-07 19:19:28 ERROR (MainThread) [custom_components.reolink_dev.base] Host 192.168.0.73 error renewing the Reolink subscription
2022-01-07 19:20:40 ERROR (MainThread) [custom_components.reolink_dev.base] Host 192.168.0.73 error renewing the Reolink subscription
2022-01-07 19:21:52 ERROR (MainThread) [custom_components.reolink_dev.base] Host 192.168.0.73 error renewing the Reolink subscription
2022-01-07 19:23:04 ERROR (MainThread) [custom_components.reolink_dev.base] Host 192.168.0.73 error renewing the Reolink subscription
2022-01-07 19:24:16 ERROR (MainThread) [custom_components.reolink_dev.base] Host 192.168.0.73 error renewing the Reolink subscription
2022-01-07 19:25:28 ERROR (MainThread) [custom_components.reolink_dev.base] Host 192.168.0.73 error renewing the Reolink subscription
```

**Comment by lsdlsd88** [comment]
same question.. thanks for your work

**Comment by MechanicalHussar** [comment]
My NVR has the same issues 5 cameras connected log full of (error 400) and sometimes when I go to entities in HA NVR will crash and reboot. Reboot is happening 2 or 3 times a day and a lost signal alarm is triggered I think I must remove this integration.

**Comment by robsymax** [comment]
Getting the same error - 2022-04-04 14:53:48 ERROR (MainThread) [custom_components.reolink_dev.base] Host 10.17.20.245 error renewing the Reolink subscription

**Comment by Bill-Dung** [comment]
I had the same error. After reinstalling an older firmware everything works fine again. I have a Reolink E1 Zoom, now with firmware 3.0.0.247 instead 3.0.0.716

**Comment by skynet01** [comment]
Any update on this? Is this related to 
reolink.exceptions.CredentialsInvalidError error that I also sometimes get?

**Comment by narsaw** [comment]
Following, same issue using standalone Reolink cams integrated into HA

**Comment by andriej** [comment]
Do you also suffer in some loss of streams (they are not always available while this occurs)?

**Comment by gettingerr** [comment]
Also have 
Subscription process ended with wrong HTTP status: 400: Bad Request
and 
Host 192.xxx.xxx.xxx error renewing the Reolink subscription

**Comment by Electronlibre2012** [comment]
hello all

error renewing the Reolink subscription : i am in same boat since 1 or 2 weeks, my motions sensor doesnt work anymore in HA.

I have uninstall integrations of cameras and Addon in HACS, then reboot, install the addon in HACS and integrations again without ssl and in http (not in https) but still no luck.

Every hour the sensor state became "unavailable" and 2 minutes after state became "clear, no motion detected". Its strange, its every hours, same minutes.

i have NRV RLN8-410-E with 4 cameras installed

i upgrade the firmware to the latest and it seems to have been the beginning of problems : 

```
ModelRLN8-410
Build No.build 2110099
Hardware No.N2MB02
Config Versionv3.0.0.0
Firmware Versionv3.0.0.148_21100909
```

any solutions please?

**Comment by skynet01** [comment]
A little update on this.. I tried the new fork of this plugin and it fixes this issue along with others: https://github.com/JimStar/reolink_cctv/

**Comment by moseisleydk** [comment]
I have just set up 3 Reolink Cam 510 Wifi.

The HA logs are flooded:

2023-03-15T18:30:21.107704375Z 2023-03-15 19:30:21.107 ERROR (MainThread) [custom_components.reolink_dev.base] Host 10.0.0.108 error renewing the Reolink subscription
2023-03-15T18:30:21.111840513Z 2023-03-15 19:30:21.111 ERROR (MainThread) [custom_components.reolink_dev.base] Host 10.0.0.176 error renewing the Reolink subscription
2023-03-15T18:30:22.434738850Z 2023-03-15 19:30:22.430 ERROR (MainThread) [custom_components.reolink_dev.base] Host 10.0.0.191 error renewing the Reolink subscription

So its for all 3 cams. I am not using NVR, just the Reolink IP Camema integration..

All sensors are just flipping between "Unavailable" and "Cleared (no detected)"
